src/app/login/login.component.ts

```
@@ -102,7 +106,9 @@
       .subscribe(res => {
         if (res.statusCode == '200') {
           if (res.data.previlegeObj && res.data.previlegeObj[0]) {
-            localStorage.setItem('loginDataResponse', JSON.stringify(res.data));
+            this.cookieService.set('Jwttoken', res.data.Jwttoken);
+            delete res.data.Jwttoken;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)